source/config/configSpec.py

```diff
@@ -192,6 +192,9 @@
 
 [upgrade]
 	newLaptopKeyboardLayout = boolean(default=false)
+
+[editableText]
+	caretMoveTimeout = integer(min=0, max=2000, default=100)

```

Comment:
I would be tempted to mention the units as part of the name for this config option, something like `caretMoveTimeoutMS` or `caretMoveTimeoutMilliSec`